### ISSUE TITLE
Comment warning

### ISSUE THREAD

**Issue by cmckni3** [sasstools/sass-lint#1136]
What version of Sass Lint are you using? 1.11.1

What did you do? Please include the actual source code causing the issue.

```scss
.next-month {
    /*!
    /*rtl:begin:ignore*//*
    */
    right: 0;
    /*!
    /*rtl:end:ignore*//*
    */
}
```

What did you expect to happen? I would like to see a warning. It might not be possible. I know it's a developer error but would be nice to detect.

What actually happened? No errors or warnings. Styles following it in the compiled css are commented out.

If you're using a IDE plugin have you tried the CLI too? Using CLI and Visual Studio Code.

**Comment by DanPurdy** [comment]
Hmm, have you got any other examples? This seems to compile as expected. Would it be a rule to pick up the use of comments or specific patterns like above?

**Comment by cmckni3** [comment]
I don't have another example. It compiles but if you have styles below it, they are commented out.

I can make an example in plunkr or codepen to demonstrate.

**Comment by cmckni3** [comment]
Huh, looks like it was very specific to Angular, Angular CLI, and AOT. I can't reproduce it in newer versions of Angular.
